extensions/ui/org.eclipse.smarthome.ui.basic/README.md

```diff
@@ -24,8 +24,18 @@ org.eclipse.smarthome.basicui:enableIcons=true
 # Default: PNG
 org.eclipse.smarthome.basicui:iconType=svg
 ```
+### Accessing Sitemaps:
 
-Screenshots:
+The Basic UI has a default layout showing all things and their corresponding items. You may create your own sitemaps and access them through the basic UI in 2 ways.
+
+1. Set the default sitemap via the Paper UI via Configuration -> Services -> Basic UI -> Configure, and set the Default Sitemap name.
+
+2. Passing the "sitemap" parameter to the URL used to access the OpenHAB server.
```

Comment:
remove "openHAB" here (note that this is documentation of ESH and thus should be independent of any specific solution)